ginga/AstroImage.py

```diff
@@ -142,7 +142,7 @@ def load_file(self, filepath, numhdu=None, naxispath=None):
             if '.' in name:
                 name = name[:name.rindex('.')]
             if numhdu is not None:
-                name += ('[%d]' % numhdu)
+                name += '[{0}]'.format(numhdu)
             self.set(name=name)
 
         self.set(path=filepath, idx=numhdu)
```

Comment:
Not sure, but I think there might be some places where `idx` is assumed to be an integer.  I'll dig around a little bit.  Also, does this work for `fitsio`-based loader?